style.css

```diff
@@ -41,9 +65,10 @@ div.navbar {
 }*/
 
 footer {
-   margin-top: 50px;
+   margin-top: 550px;
```

Comment:
It's better to use REM instead of px. If you're not using sass to compile you can calculate your rem by doing
px/16. (16 is the standard font-size browsers use so most browsers render 16px as 1rem)